src/cobalt.rs

```diff
@@ -262,11 +268,29 @@ pub fn build(config: &Config) -> Result<()> {
                 }
             }
             let src_file = source.join(file_path.as_path());
-            let dest_file = dest.join(file_path);
 
-            fs::copy(src_file.as_path(), dest_file.as_path())
-                .map_err(|e| format!("Could not copy {:?} into {:?}: {}", src_file, dest_file, e))?;
-            debug!("Copied {:?} to {:?}", src_file, dest_file);
+            if file_path.extension().unwrap_or_else(|| OsStr::new("")) == "scss" {
+                let content = sass_compile_file(src_file.as_path(), sass_opts.clone())?;
+                let mut dest_file = dest.join(file_path.clone());
+                dest_file.set_extension("css");
```

Comment:
I'm confused.  How is the path `tests/fixtures/sass/_sass/_nested.scss` being converted into the path `tests/target/sass/style/nest.css`?

I'm not seeing you change the folder nor remove the leading `_`.

Reply:
from my understand `_sass` is in the default ignore pattern (because of it's underscore), thus being excluded from copy and converting. `_sass/_nested` (similar to `_layout/main`) shall never be converted directly.

The "conversion" happens within the scss-compiling through the `@import`-directive. In `style/nest.scss` it is importing `_nested`, which in turn has an import for the other import file - both can be and should be found in the `_sass`-folder (the entry point for all imports).

So the only converting that happens here is `style/nest.scss` into `style/nest.css` (setting the extension), which internally imports nested and others from `_sass`.

Reply:
Oh, I missed the existence of ` tests/fixtures/sass/style/nest.scss`.  That makes sense now.